package.json

```diff
@@ -17,24 +17,40 @@
-    "scss-lint": "bundle exec scss-lint --config scss/.scss-lint.yml scss/*.scss",
-    "scss-lint-docs": "bundle exec scss-lint --config scss/.scss-lint.yml --exclude docs/assets/scss/docs.scss docs/assets/scss/*.scss",
-    "uglify": "uglifyjs --compress warnings=false --mangle --comments '/^!/' --output dist/js/bootstrap.min.js dist/js/bootstrap.js",
-    "uglify-docs": "uglifyjs --compress warnings=false --mangle --comments '/^!/' --output docs/assets/js/docs.min.js docs/assets/js/vendor/anchor.min.js docs/assets/js/vendor/clipboard.min.js docs/assets/js/vendor/holder.min.js docs/assets/js/src/application.js",
-    "update-shrinkwrap": "npm shrinkwrap --dev && shx mv ./npm-shrinkwrap.json ./grunt/npm-shrinkwrap.json",
-    "test": "npm run eslint && grunt test"
+    "bash": "bash",
+    "css": "npm-run-all css-lint css-compile css-prefix css-minify",
+    "css-docs": "npm-run-all css-lint-docs css-compile-docs css-prefix-docs css-minify-docs",
+    "css-lint": "bundle exec scss-lint --config scss/.scss-lint.yml scss/*.scss",
+    "css-lint-docs": "bundle exec scss-lint --config scss/.scss-lint.yml --exclude docs/assets/scss/docs.scss docs/assets/scss/*.scss",
+    "css-compile": "node-sass --output-style expanded --source-map true --precision 6 scss/bootstrap.scss dist/css/bootstrap.css && node-sass --output-style expanded --source-map true --precision 6 scss/bootstrap-grid.scss dist/css/bootstrap-grid.css && node-sass --output-style expanded --source-map true --precision 6 scss/bootstrap-reboot.scss dist/css/bootstrap-reboot.css",
+    "css-compile-docs": "node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.min.css",
+    "css-prefix": "postcss --config build/ --replace dist/css/*.css",
+    "css-prefix-docs": "postcss --config build/ --no-map --replace docs/assets/css/docs.min.css docs/examples/**/*.css",
+    "css-minify": "cleancss --level 1 --source-map --output dist/css/bootstrap.min.css dist/css/bootstrap.css && cleancss --level 1 --source-map --output dist/css/bootstrap-grid.min.css dist/css/bootstrap-grid.css && cleancss --level 1 --source-map --output dist/css/bootstrap-reboot.min.css dist/css/bootstrap-reboot.css",
+    "css-minify-docs": "cleancss --level 1 --source-map --output docs/assets/css/docs.min.css docs/assets/css/docs.min.css",
+    "js": "npm-run-all js-compile js-minify",
+    "js-docs": "npm-run-all js-lint-docs js-minify-docs",
+    "js-lint": "eslint js/ && eslint --config js/tests/.eslintrc.json --env node build/",
+    "js-lint-docs": "eslint --config js/tests/.eslintrc.json docs/assets/js/",
+    "js-compile": "npm-run-all --parallel js-compile-*",
+    "js-compile-bundle": "shx cat js/src/util.js 'js/src/!(util|popover).js' js/src/popover.js | shx sed 's/^(import|export).*//' | babel --filename js/src/bootstrap.js | node build/stamp.js > dist/js/bootstrap.js",
```

Comment:
Hi @bardiharborow, 

I have an error when this task is runned see : 

![image](https://cloud.githubusercontent.com/assets/1689750/25199218/57869926-254a-11e7-8635-b53df776bbea.png)

I know my screenshot is in french, but it said : 
`'popover).js'' isn't known at an intern or extern command`